relnotes/reopensocket.feature.md

```diff
@@ -0,0 +1,20 @@
+* `ocean.util.app.DaemonApp`
+
+  `DaemonApp`'s `ReopenableFilesExt` now supports reopening commands from the
+  UnixSocketExt. If you have the UnixSocketExt bound to the socket, sending
+  command `reopen_logfile file_path` will reopen the specified file, if
+  registered, and acknowledge the action with `ACK` or `ERROR` strings.
```

Comment:
I wonder if, for debugging purposes, a message should be returned with the `ERROR` too (like `ERROR <free text>`).

Reply:
I could return ERROR + the error text, no problem.